test/integration/install-variants_test.js

```diff
@@ -29,7 +31,7 @@ suite('install-variants', function() {
           assert.fail(err);
           done(err);
         }
-        runCommand(binPath, ['install', '--variants', '--offline'], {cwd: tmpPath}).then(() => {
+        runCommand(binPath, ['install', '--variants'], {cwd: tmpPath}).then(() => {
```

Comment:
Well, only except that explicitly building up an env that explicitly set env variables that start with `bower_` to the empty string seems to make offline installing work, even though I don't see any such env variables being printed: https://travis-ci.org/Polymer/polymer-cli/builds/202150101